tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -93,6 +96,22 @@ static cl::opt<bool>
     IsRegularization("s",
                      cl::desc("Regularize LLVM to be representable by SPIR-V"));
 
+using SPIRV::VersionNumber;
+
+static cl::opt<VersionNumber> MaxSPIRVVersion(
+    "spirv-max-version",
+    cl::desc("Choose maximum SPIR-V version which can be emitted"),
+    cl::values(clEnumValN(VersionNumber::SPIRV_1_0, "1.0", "SPIR-V 1.0"),
+               clEnumValN(VersionNumber::SPIRV_1_1, "1.1", "SPIR-V 1.1")),
+    cl::init(VersionNumber::MaximumVersion));
+
+static cl::opt<std::string>
```

Comment:
Should we use `cl::list` instead, like [here](https://github.com/llvm/llvm-project/blob/4f1d893f9ec33a493d9f35426254a0ce69716933/llvm/include/llvm/CodeGen/CommandFlags.inc#L38-L41)

Reply:
Done, thanks for suggestion!